### PR TITLE
fixed label test for new header section T50139

### PR DESCRIPTION
**Pull request by WMDE-Fisch**

### CODE REVIEW

**Comment by WMDE-Fisch** [comment]
plz review @thiemowmde @adrianheine

File: tests/browser/features/step_definitions/label_steps.rb [hunk 1]
```diff
@@ -90,29 +90,28 @@
...
 end
 
-Then /^Entity id should be displayed next to the label$/ do
+Then /^(.+) should be displayed as label having the ID of (.+)$/ do |value, item_handle|
   on(ItemPage) do |page|
-    expect(page.entity_id_span_element.visible?).to be true
-    expect(page.entity_id_span.include?(@item_under_test['id'])).to be true
+    expect(page.first_heading?).to be true
+    page.wait_for_label(value)
+    expect(page.first_heading_element.text).to be == value + '(' + @items[item_handle]['id'] + ')'
```
Comment (adrianheine) [inline tests/browser/features/step_definitions/label_steps.rb]: These three lines could be factored out and reused in this file.

File: tests/browser/features/step_definitions/property_steps.rb [hunk 2]
```diff
@@ -10,6 +10,6 @@
   expect(on(PropertyPage).property_datatype_heading?).to be true
 end
 
-Then /^Property datatype should display (.*)$/ do | datatype |
+Then /^Property datatype should display (.*)$/ do |datatype|
```
Comment (adrianheine) [inline tests/browser/features/step_definitions/property_steps.rb]: Is there a styleguide for this?

Comment (WMDE-Fisch) [inline tests/browser/features/step_definitions/property_steps.rb]: rubocop complained ....

    features/step_definitions/property_steps.rb:13:52: C: Space before first block parameter detected.
    Then /^Property datatype should display (.*)$/ do | datatype | 
    features/step_definitions/property_steps.rb:13:61: C: Space after last block parameter detected.
    Then /^Property datatype should display (.*)$/ do | datatype |

File: tests/browser/features/support/modules/entity_module.rb [hunk 3]
```diff
@@ -24,16 +24,15 @@ module EntityPage
   # ***** ACCESSORS *****
   # label UI
   h1(:mw_first_heading, id: 'firstHeading')
-  h1(:first_heading, css: 'h1.wb-firstHeading')
-  span(:entity_label_span, css: '.wikibase-labelview-text')
-  span(:entity_id_span, css: '.wikibase-labelview-entityid')
+  h1(:first_heading, css: 'h1.wikibase-entitytermsview-heading-label')
+  span(:entity_id_span, css: '.wikibase-entitytermsview-heading-label-id')
   text_field(:label_input_field, css: '.wikibase-labelview-text input')
```
Comment (adrianheine) [inline tests/browser/features/support/modules/entity_module.rb]: This selector still fails in features/empty_label_and_description.feature (17, 25, 31, 41, 49, 55).

File: tests/browser/features/step_definitions/property_steps.rb (hunk 2 above)
Comment (adrianheine) [inline tests/browser/features/step_definitions/property_steps.rb]: Cool, it's even automated :) How come it didn't fail before this change? Does it only check modified files?

Comment (WMDE-Fisch) [inline tests/browser/features/step_definitions/property_steps.rb]: nope it checks everything :-] - guess travis was not configured until then or the rubocup default settings changed...

File: tests/browser/features/support/modules/entity_module.rb (hunk 3 above)
Comment (WMDE-Fisch) [inline tests/browser/features/support/modules/entity_module.rb]: empty_label_and_description.feature is fixed in #41

File: tests/browser/features/step_definitions/label_steps.rb [hunk 4]
```diff
@@ -90,29 +90,28 @@
...
-    expect(@browser.title.include?(@item_under_test['label'])).to be true
-    expect(page.entity_label_span).to be == @item_under_test['label']
+    page.wait_for_label(@item_under_test['label'])
+    expect(page.first_heading_element.text).to be == @item_under_test['label'] + '(' + @item_under_test['id'] + ')'
   end
 end
 
 Then(/^Label element should be there$/) do
-  expect(on(ItemPage).entity_label_span?).to be true
+  expect(on(ItemPage).first_heading?).to be true
```
Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/label_steps.rb]: For future reference, if you are just checking if an element exists, this is probably more readable:

    expect(on(ItemPage).first_heading_element).to exist

**Pull request merged**